var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -46,30 +47,50 @@ class Neuron(Package):
     version('7.2', '5486709b6366add932e3a6d141c4f7ad')
     version('develop', git=git, commit='9f36b132ce1b5e4', preferred=True)
 
-    variant('binary',        default=True, description="Create special as a binary instead of shell script")
-    variant('coreneuron',    default=True, description="Patch hh.mod for CoreNEURON compatibility")
+    variant('binary',        default=True,  description="Create special as a binary instead of shell script")
+    variant('coreneuron',    default=True,  description="Patch hh.mod for CoreNEURON compatibility")
     variant('cross-compile', default=False, description='Build for cross-compile environment')
     variant('debug',         default=False, description='Build debug with O0')
     variant('mpi',           default=True,  description='Enable MPI parallelism')
     variant('multisend',     default=True,  description="Enable multi-send spike exchange")
     variant('profile',       default=False, description="Enable Tau profiling")
     variant('python',        default=True,  description='Enable python')
-    variant('shared',        default=True, description='Build shared libraries')
-    variant('rx3d',          default=False, description="Enable cython translated 3-d rxd")
+    variant('shared',        default=True,  description='Build shared libraries')
+    variant('pysetup',       default=True,  description="Build Python module with setup.py")
+    variant('rx3d',          default=False, description="Enable cython translated 3-d rxd. Depends on pysetup")
 
     depends_on('autoconf',   type='build')
     depends_on('automake',   type='build')
     depends_on('bison',      type='build')
     depends_on('flex',       type='build')
     depends_on('libtool',    type='build')
     depends_on('pkgconfig',  type='build')
+    depends_on('readline')
 
     depends_on('mpi',         when='+mpi')
     depends_on('ncurses',     when='~cross-compile')
     depends_on('python@2.6:', when='+python')
     depends_on('tau',         when='+profile')
 
-    conflicts('~shared', when='+python')
+    conflicts('~shared',  when='+python')
+    conflicts('+pysetup', when='~python')
+    conflicts('+rx3d',    when='~pysetup')
+
+    _default_options = ['--without-iv',
```

Comment:
Do we need  `_` in name? I would prefer to remove them. (same for `_specs_to_options`).